target/linux/lantiq/image/Makefile

```diff
@@ -744,6 +744,17 @@ define Device/avm_fritz3370-rev2-micron
 endef
 TARGET_DEVICES += avm_fritz3370-rev2-micron
 
+define Device/avm_fritz3390
```

Comment:
Note that there have been changes to lantiq recently:
- Device definitions have been moved to individual Makefiles per board.
- DTS files have been renamed by soc_vendor_model scheme.
- DEVICE_DTS is assembled automatically based on node name and to-be-specified SOC variable (in your case `SOC := vr9`)